docs/src/components/NotebookDownloadButton/index.tsx

```diff
@@ -20,6 +21,12 @@ export function NotebookDownloadButton({ children, href }: NotebookDownloadButto
         }
       }
 
+      if (!Version.includes('dev')) {
+        // Replace 'master' with the current version to pin the download to the released version
+        // and avoid 404 errors
+        href = href.replace(/\/master\//, `/v${Version}/`);
```

Comment:
Docs for `x.y.z` should reference files at `vx.y.z`. Referencing master leads to 404.

Reply:
In general, we should not referencing master since it's mutable.

Reply:
not sure who downloads these notebooks tho